var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -29,14 +29,23 @@ class Ghostscript(Package):
     """An interpreter for the PostScript language and for PDF."""
 
     homepage = "http://ghostscript.com/"
-    url = "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-9.18.tar.gz"
 
+    version('9.21', '6f60d7fcb5eef6a8bec5abedf21c6a7008a8c0c7')
     version('9.18', '33a47567d7a591c00a253caddd12a88a')
 
     parallel = False
 
     depends_on('libtiff')
 
+    def url_for_version(self, version):
```

Comment:
It won't have any impact on existing users as long as the hash hasn't changed. You should confirm that you can still download the old version and the hash is the same.

We don't need to cater to anyone who needs an older version unless we know that they exist. If anyone out there wanted an older version, they would have added it by now. If someone does want one in the future, they can update the `url_for_version` themselves.